pkg/client/selfhosted/selfhosted.go

```diff
@@ -76,105 +65,98 @@ type V1Compatibility struct {
 }
 
 func New(ctx context.Context, opts Options) (*Client, error) {
-	client := &http.Client{
-		Timeout: time.Second * 5,
+	client := &Client{
+		Client: &http.Client{
+			Timeout: time.Second * 5,
```

Comment:
I've hit this timeout about 10-20% of the time. When I changed it to 10 seconds, I had 0 issues.